corehq/apps/export/const.py

```diff
@@ -27,10 +31,12 @@
 FORM_EXPORT = 'form'
 CASE_EXPORT = 'case'
 
-MAIN_TABLE = None
+MAIN_TABLE = []
```

Comment:
Sorry, should have explained. It's follow up from here: https://github.com/dimagi/commcare-hq/pull/10182#discussion_r51935186